tests/src/AppBundle/Model/ServiceModel.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace AppBundle\Model;
+
+use PommProject\ModelManager\Model\Model;
+use PommProject\ModelManager\Model\Projection;
+use PommProject\ModelManager\Model\ModelTrait\WriteQueries;
+
+use PommProject\Foundation\Where;
+
+use AppBundle\Model\AutoStructure\Service as ServiceStructure;
+use AppBundle\Model\Service;
+
+use Psr\Log\LoggerInterface;
+
+/**
+ * ServiceModel
+ *
+ * Model class for table service.
+ *
+ * @see Model
+ */
+class ServiceModel extends Model
+{
+    use WriteQueries;
+
+    /** @var LoggerInterface */
+    private $logger;
+
+    /**
+     * __construct()
```

Comment:
putting the method name as short description does not make sense (and would even hurt readability if you generate the API doc)